server/utilities/Validation /signUp.js

```diff
@@ -1,16 +1,25 @@
 const Joi = require('joi');
 
 const signUpValidation = ({
-  email, first_name, last_name, password,
+  email, firstName, lastName, password,
 }) => {
   const schema = Joi.object({
     email: Joi.string().email().required(),
-    first_name: Joi.string().required(),
-    last_name: Joi.string().required(),
-    password: Joi.string().alphanum().min(6).required(),
+    firstName: Joi.string().required(),
+    lastName: Joi.string().required(),
```

Comment:
You can add min() and max() here for both first and last name, the name should not be too small or too long.